SYCL/ESIMD/esimd_test_utils.hpp

```diff
@@ -161,4 +162,30 @@ To bit_cast(const From &src) noexcept {
   return dst;
 }
 
+// Timer class for measuring elasped time
+class Timer {
+public:
+  Timer() : start_(std::chrono::steady_clock::now()) {}
+
+  double Elapsed() {
+    auto now = std::chrono::steady_clock::now();
+    return std::chrono::duration_cast<Duration>(now - start_).count();
+  }
+
+private:
+  using Duration = std::chrono::duration<double>;
+  std::chrono::steady_clock::time_point start_;
+};
+
+// return msecond
+inline double report_time(const std::string &msg, event e0, event en) {
```

Comment:
Maybe change the name to more descriptive like `get_elapsed_time`? 
I think it also makes sense to change the order of arguments: `event e0, event e1, const std::string &msg`.
Do we need two different event arguments? I see we pass the same event object every time.
And please add a description for the function.

Reply:
I think the name is fine. I have added some comment for report_time. two-event usage is in BitonicSortK and BitonicSortKv2

Reply:
The `report_time` function does not report anything, so to me, the name looks confusing. Anyway, it should not block the PR.